pkgs/development/python-modules/rtp/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3
+
+  # nativeCheckInputs
+, hypothesis
+, unittestCheckHook
+
+  # propagatedBuildInputs
+, six
+, flask
+}:
+
+buildPythonPackage rec {
+  pname = "rtp";
+  version = "0.0.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-xUfI8lQD/EhA4NrE4y+KHlie+wwBWkYkDqFLVWKwOzA=";
+  };
+
+  # We don't run mypy tests, so we don't need the dependency
+  postPatch = ''
+    sed -i '/"mypy"/d' setup.py
+  '';
+
+  nativeCheckInputs = [
+    hypothesis
+    unittestCheckHook
+  ];
+
+  unittestFlagsArray = [ "-s" "tests" "-v" ];
+
+  propagatedBuildInputs = [
+    six
+    flask
```

Comment:
Are these two used at all?
Maybe we should file an upstream PR removing all three from `install_requires `.

Reply:
Fair point. Didn't thought of that... But yes, they are indeed since 4y in the setup.py but are not used. I will take care of this

Reply:
Upstream PR: https://github.com/bbc/rd-apmm-python-lib-rtp/pull/10